File: C++/oop.cpp
#include <iostream>

using namespace std;

class Car {
public:
    string tenxe;
    string mausac;
    Car(string tenxe, string mausac){
        this->tenxe = tenxe;
        this->mausac = mausac;
    }
    string dungxe(string mucdich){
        return this->tenxe + "\t dang dung xe de " + mucdich;
    }
};

int main(){
    Car toyota("toyota", "den");
    cout<<toyota.dungxe("do xang");
}

File: Typescript/oop.ts
export class Car {
	tenxe:string;
	mausac:string;
	constructor(tenxe:string, mausac:string) {
		this.tenxe = tenxe;
		this.mausac = mausac;
	}
	public dungxe(mucdich:string) {
		return `${this.tenxe}\t dang dung xe de ${mucdich}`		
	}
}

let toyota = new Car("toyota", "den")

console.log(toyota.dungxe("do xang"))